gufe/components/smallmoleculecomponent.py

```diff
@@ -209,7 +209,10 @@ def to_openff(self):
         """
         from openff.toolkit.topology import Molecule as OFFMolecule
 
-        m = OFFMolecule(self._rdkit, allow_undefined_stereo=True)
+        m = OFFMolecule(self._rdkit,
```

Comment:
I think we need to change this so that it calls `Molecule.from_rdkit`. Do we know if there are any reason why we were building the molelcule directly through `__init__` instead of just calling the classmethod?

Reply:
Done! Thanks @IAlibay!